google-cloud-bigquery-data_transfer/docs/toc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "guides": [],
+  "services": [
+    {
+      "title": "Bigquery DataTransfer",
+      "type": "google/cloud/bigquery/data_transfer",
```

Comment:
@landrito These `type` values are consumed by the Angular site app and should not have underscores:  `datatransfer`, not `data_transfer`.

Reply:
Done.